<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>617</width>
    <height>410</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Valve Control</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <layout class="QVBoxLayout" name="verticalLayout_2">
    <item>
     <widget class="QGroupBox" name="groupBoxValveControl">
      <property name="title">
       <string>Valve Control</string>
      </property>
      <layout class="QHBoxLayout" name="horizontalLayout_4">
       <item>
        <widget class="QFrame" name="frameValveControl">
         <property name="frameShape">
          <enum>QFrame::NoFrame</enum>
         </property>
         <property name="frameShadow">
          <enum>QFrame::Raised</enum>
         </property>
         <property name="lineWidth">
          <number>1</number>
         </property>
        </widget>
       </item>
      </layout>
     </widget>
    </item>
    <item>
     <widget class="QGroupBox" name="groupBoxCycle">
      <property name="sizePolicy">
       <sizepolicy hsizetype="Preferred" vsizetype="Minimum">
        <horstretch>0</horstretch>
        <verstretch>0</verstretch>
       </sizepolicy>
      </property>
      <property name="title">
       <string>Programmable Cycle</string>
      </property>
      <layout class="QVBoxLayout" name="verticalLayout">
       <item>
        <layout class="QHBoxLayout" name="horizontalLayout_2">
         <item>
          <widget class="QLabel" name="label">
           <property name="text">
            <string>Program:</string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QLineEdit" name="lineEditProgram">
           <property name="readOnly">
            <bool>true</bool>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QToolButton" name="toolButtonProgram">
           <property name="text">
            <string>...</string>
           </property>
          </widget>
         </item>
        </layout>
       </item>
       <item>
        <layout class="QHBoxLayout" name="horizontalLayout">
         <property name="sizeConstraint">
          <enum>QLayout::SetNoConstraint</enum>
         </property>
         <item>
          <widget class="QPushButton" name="pushButtonStart">
           <property name="sizePolicy">
            <sizepolicy hsizetype="Minimum" vsizetype="Preferred">
             <horstretch>0</horstretch>
             <verstretch>0</verstretch>
            </sizepolicy>
           </property>
           <property name="text">
            <string>Start</string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QPushButton" name="pushButtonStop">
           <property name="sizePolicy">
            <sizepolicy hsizetype="Minimum" vsizetype="Preferred">
             <horstretch>0</horstretch>
             <verstretch>0</verstretch>
            </sizepolicy>
           </property>
           <property name="text">
            <string>Stop</string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QLabel" name="label_2">
           <property name="text">
            <string>Cycles:</string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QSpinBox" name="spinBoxCycles">
           <property name="sizePolicy">
            <sizepolicy hsizetype="Minimum" vsizetype="Preferred">
             <horstretch>0</horstretch>
             <verstretch>0</verstretch>
            </sizepolicy>
           </property>
           <property name="accelerated">
            <bool>true</bool>
           </property>
           <property name="maximum">
            <number>2147483647</number>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QLabel" name="label_3">
           <property name="text">
            <string>Phase interval (ms):</string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QSpinBox" name="spinBoxIntervalMillis">
           <property name="sizePolicy">
            <sizepolicy hsizetype="Minimum" vsizetype="Preferred">
             <horstretch>0</horstretch>
             <verstretch>0</verstretch>
            </sizepolicy>
           </property>
           <property name="accelerated">
            <bool>true</bool>
           </property>
           <property name="maximum">
            <number>2147483647</number>
           </property>
           <property name="singleStep">
            <number>10</number>
           </property>
           <property name="value">
            <number>100</number>
           </property>
          </widget>
         </item>
        </layout>
       </item>
       <item>
        <layout class="QHBoxLayout" name="horizontalLayout_3">
         <item>
          <widget class="QLabel" name="label_4">
           <property name="text">
            <string>Built-in Program:</string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QComboBox" name="comboBoxBuiltIn">
           <item>
            <property name="text">
             <string>Toggle Valve</string>
            </property>
           </item>
           <item>
            <property name="text">
             <string>5-Phase Pump</string>
            </property>
           </item>
          </widget>
         </item>
         <item>
          <widget class="QPushButton" name="pushButtonLoadBuiltIn">
           <property name="enabled">
            <bool>true</bool>
           </property>
           <property name="text">
            <string>Load</string>
           </property>
           <property name="checkable">
            <bool>false</bool>
           </property>
          </widget>
         </item>
        </layout>
       </item>
      </layout>
     </widget>
    </item>
   </layout>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>617</width>
     <height>22</height>
    </rect>
   </property>
   <widget class="QMenu" name="menuFile">
    <property name="title">
     <string>File</string>
    </property>
    <addaction name="actionOpen"/>
    <addaction name="actionClose"/>
   </widget>
   <widget class="QMenu" name="menuDevice">
    <property name="enabled">
     <bool>true</bool>
    </property>
    <property name="title">
     <string>Device</string>
    </property>
    <widget class="QMenu" name="menuShift_Register">
     <property name="title">
      <string>Shift Register</string>
     </property>
     <addaction name="actionSR1"/>
     <addaction name="actionSR2"/>
     <addaction name="actionSR3"/>
     <addaction name="actionSR4"/>
     <addaction name="actionSR5"/>
     <addaction name="actionSR6"/>
    </widget>
    <widget class="QMenu" name="menuSerial">
     <property name="title">
      <string>Serial</string>
     </property>
     <addaction name="actionReset_Input"/>
     <addaction name="actionReset_Output"/>
    </widget>
    <addaction name="actionRestart_Device"/>
    <addaction name="actionTurn_Off_All"/>
    <addaction name="separator"/>
    <addaction name="actionStart"/>
    <addaction name="actionStop"/>
    <addaction name="separator"/>
    <addaction name="menuShift_Register"/>
    <addaction name="separator"/>
    <addaction name="menuSerial"/>
   </widget>
   <widget class="QMenu" name="menuHelp">
    <property name="title">
     <string>Help</string>
    </property>
    <addaction name="actionDocumentation"/>
    <addaction name="actionUpdate"/>
   </widget>
   <addaction name="menuFile"/>
   <addaction name="menuDevice"/>
   <addaction name="menuHelp"/>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
  <action name="actionOpen">
   <property name="text">
    <string>Open...</string>
   </property>
  </action>
  <action name="actionClose">
   <property name="text">
    <string>Close</string>
   </property>
  </action>
  <action name="actionMax_Operations">
   <property name="text">
    <string>Max Operations</string>
   </property>
  </action>
  <action name="actionMax_Phases">
   <property name="text">
    <string>Max Phases</string>
   </property>
  </action>
  <action name="actionProgrammable_Cycle">
   <property name="text">
    <string>Programmable Cycle</string>
   </property>
  </action>
  <action name="actionDocumentation">
   <property name="text">
    <string>Documentation</string>
   </property>
  </action>
  <action name="actionRestart_Device">
   <property name="enabled">
    <bool>true</bool>
   </property>
   <property name="text">
    <string>Restart Arduino</string>
   </property>
  </action>
  <action name="actionTurn_Off_All">
   <property name="text">
    <string>Turn Off All</string>
   </property>
  </action>
  <action name="actionStart">
   <property name="text">
    <string>Start</string>
   </property>
  </action>
  <action name="actionStop">
   <property name="text">
    <string>Stop</string>
   </property>
  </action>
  <action name="actionUsb0">
   <property name="text">
    <string>usb0</string>
   </property>
  </action>
  <action name="actionSR1">
   <property name="checkable">
    <bool>true</bool>
   </property>
   <property name="checked">
    <bool>false</bool>
   </property>
   <property name="text">
    <string>1</string>
   </property>
  </action>
  <action name="actionSR2">
   <property name="checkable">
    <bool>true</bool>
   </property>
   <property name="text">
    <string>2</string>
   </property>
  </action>
  <action name="actionSR3">
   <property name="checkable">
    <bool>true</bool>
   </property>
   <property name="text">
    <string>3</string>
   </property>
  </action>
  <action name="actionSR4">
   <property name="checkable">
    <bool>true</bool>
   </property>
   <property name="text">
    <string>4</string>
   </property>
  </action>
  <action name="actionSR5">
   <property name="checkable">
    <bool>true</bool>
   </property>
   <property name="text">
    <string>5</string>
   </property>
  </action>
  <action name="actionSR6">
   <property name="checkable">
    <bool>true</bool>
   </property>
   <property name="text">
    <string>6</string>
   </property>
  </action>
  <action name="action7">
   <property name="checkable">
    <bool>true</bool>
   </property>
   <property name="text">
    <string>7</string>
   </property>
  </action>
  <action name="action8">
   <property name="checkable">
    <bool>true</bool>
   </property>
   <property name="text">
    <string>8</string>
   </property>
  </action>
  <action name="actionAsdf">
   <property name="text">
    <string>asdf</string>
   </property>
  </action>
  <action name="actionReset_Input">
   <property name="text">
    <string>Reset Input</string>
   </property>
  </action>
  <action name="actionReset_Output">
   <property name="text">
    <string>Reset Output</string>
   </property>
  </action>
  <action name="actionUpdate">
   <property name="text">
    <string>Update</string>
   </property>
  </action>
 </widget>
 <resources/>
 <connections/>
</ui>
